### DISCUSSION TITLE
Add a property to skip via command line

### DISCUSSION THREAD

**Discussion by Thihup** [wiverson/jtoolprovider-plugin#14] [General]
Currently, the only way to skip this plugin is by adding the <skip>true</skip>.
It would be great to have a way to skip the execution when running Maven, something like `mvn package -Djtoolprovider.skip`

**Comment by wiverson** [comment]
Right now the most common usage of this plugin AFAIK is in projects like jpackage-template, which has many executions. Skipping the entire plugin run would lead to confusing results.

In lieu of that, I would suggest [using profiles instead](https://www.baeldung.com/maven-profiles). Then you can target specific executions to skip however you would like.
